src/app/services/networkManagerService/networkManager.spec.js

```diff
@@ -15,37 +15,68 @@ describe('network Manager service', () => {
 
   beforeEach(angular.mock.module('ngCordova.plugins.network'));
 
-  beforeEach(inject((_$cordovaNetwork_, _networkManagerService_) => {
+  beforeEach(inject((_$cordovaNetwork_, _networkManagerService_, _$window_) => {
     $cordovaNetwork = _$cordovaNetwork_;
     networkManagerService = _networkManagerService_;
-
-    window.Connection = {
+    $window = _$window_;
+    navigator = $window.navigator;
+    $window.Connection = {
       UNKNOWN: 'UNKNOWN',
       NONE: 'NONE',
       CELL_2G: 'CELL_2G'
     };
-
-    navigator.connection = {
-      type: window.Connection.NONE
-    };
+    navigator.connection = { type: $window.Connection.NONE};
   }));
 
-  it('should return isOnline', () => {
-    navigator.connection.type = window.Connection.CELL_2G;
-    expect($cordovaNetwork.isOnline()).to.be.true;
+  it('should be online', () => {
```

Comment:
Viendo el nombre más descriptivo que utilizamos para los siguientes tests, también cambiaría este a algo rollo "should be online when not a webview and has connection"

Reply:
ok